src/modules/billing/service/charge-processor/mappers.js

```diff
@@ -0,0 +1,4 @@
+const { mapKeys, isObject } = require('lodash');
+const camelCase = require('camelcase');
+
+exports.camelCaseKeys = obj => isObject(obj) && mapKeys(obj, (value, key) => camelCase(key));
```

Comment:
Could this be removed and the code at src/lib/camel-case-keys.js be used?

Reply:
This should be fixed